<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manage_Facebook_Account</class>
 <widget class="QMainWindow" name="Manage_Facebook_Account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Facebook_Account</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>filler</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Filter by uid</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="checkBox_filter_uid">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_filter_uid">
            <property name="placeholderText">
             <string>enter the uid</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Filter by name</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QCheckBox" name="checkBox_filter_name">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_filter_name">
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Filter by State</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkBox_filter_state">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_filter_state">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_reset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea_list_facebook_account">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>243</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_list_facebook_account">
        <item>
         <widget class="QTableWidget" name="tableWidget_list_account">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_statistics">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>statistics</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QScrollArea" name="scrollArea_state">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_statistics">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>760</width>
            <height>165</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_statistics"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_share_permissions">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>share permissions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_remove_permissions">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>remove permissions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Reload">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Reload</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_result_api">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigs">
    <property name="title">
     <string>Configs</string>
    </property>
    <addaction name="actionConfigs_Authentication"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionImport_Cookie"/>
    <addaction name="actionProxy_Manager"/>
   </widget>
   <addaction name="menuConfigs"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConfigs_Authentication">
   <property name="text">
    <string>Configs Authentication</string>
   </property>
  </action>
  <action name="actionImport_Cookie">
   <property name="text">
    <string>Import Cookie</string>
   </property>
  </action>
  <action name="actionProxy_Manager">
   <property name="text">
    <string>Proxy Manager</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
